setup.py

```diff
@@ -29,7 +29,7 @@ class pyk4a_build_ext(build_ext):
     boolean_options = build_ext.boolean_options + ['enable-body-tracking', ]
 
     def initialize_options(self):
-        self.enable_body_tracking = False
+        self.enable_body_tracking = True
```

Comment:
The plan was to merge the body tracking branch into the master branch someday.
I worry we will forget to reset this to False before the merge.